src/equations/numerical_fluxes.jl

```diff
@@ -2,16 +2,16 @@
 # This file contains general numerical fluxes that are not specific to certain equations
 
 """
-    flux_central(u_ll, u_rr, orientation, equations::AbstractEquations)
+    flux_central(u_ll, u_rr, orientation_or_normal, equations::AbstractEquations)
```

Comment:
Passing a normal here as `orientation` bothered us all the time, but we couldn't find a better word for it. That's a really nice solution.